source/content/guides/localdev/04-troubleshoot-support.md

```diff
@@ -48,6 +48,19 @@ The contents of `wp-contents/uploads` should be symlinked to files, in keeping w
 
 You can [contact support](#contact-support) for help moving your files out of the codebase.
 
+### Clone of a New Site Failed
+
+If you attempt to clone a site before opening the admin panel to initialize the database, Localdev will fail to pull it locally. The logs under **show detail (beta)** will show:
```

Comment:
```suggestion
If you attempt to initialize a site within Localdev before you've completed the CMS install on Pantheon, it will fail when attempting to pull the (non-existent) database from the platform. The logs under **show detail (beta)** will show:
```